spec/views/users/edit.html.erb_spec.rb

```diff
@@ -77,10 +77,6 @@
     it "shows no organisation set" do
       expect(rendered).to have_text("No organisation set")
     end
-
```

Comment:
We lose the select an organisation text - we could add something similar back, either by calling the blank option "select an organisation" or by using placeholder text. I'm not sure either is recommended by the autocomplete component though?